### DISCUSSION TITLE
Keefe's Flute

### DISCUSSION THREAD

**Discussion by Sonoton** [edwardkort/WWIDesigner#140] [Flute Study] [answered]
I am looking to try to get a standard conceptual flute to serve as a reference.  Keefe's flute is very well documented (it is described in Lefebvre's dissertation).  When I put the exact dimensions of Keefe's flute in WWD in centimeters, and using the subtraction of an excess 42 millimeters that Lefebvre mentions, along with using beta factor 0.35, emb. hole height 0.340 , airstream length 0.40), the best deviation that is obtained is over 250 cents on every note.  When I slightly reoptimize the flute length and hole positions by hand, I can get a net error of zero and a deviation of 0.48 cents over all the six notes (which I would say is a success).  This reoptimization is a lengthening of the flute by 8 millimeters and a displacement of the toneholes towards the bell end by 4 millimeters.  It seems that these corrections although small are somewhat greater than what one would expect, as I trust that the authors are pretty careful in what they publish.  I am wondering if there is still some unfortunate randomness in the process of flute design that I have not considered.  Any guidance you might suggest would be appreciated.  Thank you for your time
[KeefesFlute.zip](https://github.com/edwardkort/WWIDesigner/files/7232384/KeefesFlute.zip)
[TrueReconciledKeefeFlute.zip](https://github.com/edwardkort/WWIDesigner/files/7232386/TrueReconciledKeefeFlute.zip)
[KeefeTuning-D4-Equal.zip](https://github.com/edwardkort/WWIDesigner/files/7232388/KeefeTuning-D4-Equal.zip)

**Comment by bhp1** [comment, marked as answer]
You can ignore the additional 42 mm for the excitation mechanism; WIDesigner incorporates its own excitation model.  The reference point is the centre of the embouchure hole.  This gives us this instrument geometry:
[KeefesFlute.zip](https://github.com/edwardkort/WWIDesigner/files/7240328/KeefesFlute.zip)

On the WIDesigner options, set the air temperature to 27.0 C and the blowing level (on the Whistle and Flute tab) to zero.  Display the tuning graph, and you should see a neat, nearly straight, line of green dots across the notes of the first octave (excluding, of course, C natural).  That nearly straight line means it would be very easy to play those notes at those pitches.  With the parameters I used, including an air temperature of 27 C, blowing level zero, and zero weights on every note except the diatonic scale from D4 through D5 (including D5), I was able to get the breath curve, the line of blue circles, very close to the line of green dots, so the tuning table predicted every note except D5 was within 5 cents of its target.  In this case, the tuning table doesn't really matter much; what matters is that neat, nearly straight, line of green dots, which tells us that _there exists_ a simple breath curve that will easily play that flute in tune, whether or not WIDesigner has a breath curve in its library that neatly matches it.
![KeefeTuningTable](https://user-images.githubusercontent.com/1645842/135010039-ade06189-37b2-46bf-a076-5926fd7585d6.png)
